Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -384,7 +360,7 @@ data_to_plot2$TRTACT <- factor(data_to_plot2$TRTACT,
 
```

Comment:
lines 347 through 362 can be deleted, and line 363 replaced with:
gg <- ggplot(data = pkpd_data_wide, aes(x = CONC, y = RESPONSE, color = TIME))